### ISSUE TITLE
Mockery 1.6.0 on PHP 7.3 - parse error DefinedTargetClass.php:53

### ISSUE THREAD

**Issue by michalbundyra** [mockery/mockery#1266]
```
ParseError: syntax error, unexpected '...' (T_ELLIPSIS), expecting ']'

[...]/vendor/mockery/mockery/library/Mockery/Generator/DefinedTargetClass.php:53
```

[Unpacking inside arrays](https://www.php.net/manual/en/migration74.new-features.php#migration74.new-features.core.unpack-inside-array) is available since PHP 7.4

**Comment by davedevelopment** [comment]
Fuck.

Sorry everyone, I've not been paying much attention lately. Will install 7.3 and see how quickly I can get it running.

**Comment by davedevelopment** [comment]
I can't build 7.3 right now, probably openssl mismatches.

**Comment by davedevelopment** [comment]
I don't really want to re-tag like it never happened. Can I tell packagist to ignore 1.6.0 and then tag 1.6.1 with the `^7.4 || ^8.0` constraint?

**Comment by GrahamCampbell** [comment]
> Can I tell packagist to ignore 1.6.0

No. It'll get re-synced as long as it exists on GitHub. You could delete the tag from both?

**Comment by anomiex** [comment]
We're running into this too on our CI runs with PHP 7.3.

If you delete the 1.6.0 tag from the repo, packagist should delist it. If not, you should be able to log in to packagist and manually delete it.

BTW, the problematic commit seems to be e0ecc277b14ec42535db70fa2f8ad2e44db01e86, and it could probably be fixed by changing it to something like
```php
        return array_unique(array_merge(['\AllowDynamicProperties'], array_map(
            static fn (ReflectionAttribute $attribute): string => '\\' . $attribute->getName(),
            $this->rfc->getAttributes()
        )));
```

**Comment by GrahamCampbell** [comment]
> If you delete the 1.6.0 tag from the repo, packagist should delist it. If not, you should be able to log in to packagist and manually delete it.

I think you will need to manually delete it.

**Comment by davedevelopment** [comment]
I think that's sorted :facepalm: 

https://github.com/mockery/mockery/releases/tag/1.6.1

Again, apologies everyone.

**Comment by anomiex** [comment]
Confirmed that our PHP 7.3 runs in CI are picking up 1.5.1 now and so are working again. Thanks!

**Comment by michalbundyra** [comment]
@davedevelopment thanks, works for me.

**Comment by anomiex** [comment]
It looks like the broken 1.6.0 tag has reappeared on Packagist, breaking our CI again.

**Comment by GrahamCampbell** [comment]
Looks like someone who has the tag locally has re-pushed it.

**Comment by GrahamCampbell** [comment]
cc @ghostwriter

**Comment by davedevelopment** [comment]
Deleted here and on packagist

**Comment by ghostwriter** [comment]
@GrahamCampbell That was my mistake,

Thanks for the heads up.

@davedevelopment Thanks for taking care of that.
